taipy/core/config/job_config.py

```diff
@@ -13,70 +15,63 @@ class JobConfig:
     Parameters:
         mode (str): The Taipy operating mode. By default, the "standalone" mode is set. On Taipy enterprise,
             the "airflow" mode is available.
-        nb_of_workers (int): The maximum number of running workers to execute jobs. It must be a positive integer.
-            The default value is 1.
```

Comment:
Since JobConfig is exposed, I believe we want to keep this documentation.

Reply:
It is now belong to StandaloneConfig

Reply:
I would keep this documentation. I would also keep the nb_of_worker parameter as Optional with a default value =1. And I would explicitly document that it is used only in standalone mode.

The purpose of this is to keep autocompletion for the basic usage of this method, which is the standalone mode.